### PR TITLE
Create sailesh.json

### PR DESCRIPTION
**Pull request by saileshbro**

### CODE REVIEW

File: domains/sailesh.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Sailesh's Personal Website",
+  "owner": {
+    "username": "saileshbro",
+    "email": "saileshbro@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://saileshbro.github.io/saileshbro/",
```
Comment (abhinavdalal) [inline domains/sailesh.json:8]: @saileshbro 
CNAME needs to be saileshbro.github.io
you may want to add it as URL
```suggestion
    "URL": "https://saileshbro.github.io/saileshbro/",
```

**Review by abhinavdalal — CHANGES_REQUESTED** [review]

**Comment by saileshbro** [comment]
Can anyone review this?

File: domains/sailesh.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Sailesh's Personal Website",
+  "owner": {
+    "username": "saileshbro",
+    "email": "saileshbro@gmail.com"
+  },
+  "record": {
+    "CNAME": "saileshbro.github.io",
+    "URL": "https://saileshbro.github.io/saileshbro/"
```
Comment (phenax) [inline domains/sailesh.json:9]: You cannot have a cname and url redirection together this way. It should either be a cname record to your github pages link or a url redirection.

```suggestion
    "CNAME": "saileshbro.github.io"
```

**Review by phenax — CHANGES_REQUESTED** [review]

**Comment by saileshbro** [comment]
@phenax Is it okay now?

**Review by phenax — APPROVED** [review]

**Pull request merged**